dist/OutSystemsUI.d.ts

```diff
@@ -209,6 +210,27 @@ declare namespace OSFramework.OSUI.GlobalEnum {
         TopLeft = "top-left",
         TopRight = "top-right"
     }
+    enum FloatingAlignment {
```

Comment:
@BenOsodrac Shouldn't we have a similar approach as we have for the other providers, where we have access to the version we're using?
Example:
`enum ProviderInfo {
        Name = "VirtualSelect",
        Version = "1.0.37"
    }`

Reply:
Done, also added a ReadMe!